src/ProjectTemplates/Web.ProjectTemplates/content/Grpc-WebApplication/Program.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Company.WebApplication1
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.ConfigureKestrel(options =>
+                    {
+                        options.Limits.MinRequestBodyDataRate = null;
```

Comment:
Known issue?

Reply:
https://github.com/grpc/grpc-dotnet/issues/15

Would appreciate your insight in the best way to handle this 😬

Reply:
On this note, the line here is only required for client streaming and duplex streaming scenarios. Since the sample only does unary calls, we could remove this line. However, users will run into a nasty connection resets due to rate limits which is why I originally added this.

Reply:
Blah, the thing I was going to suggest only works for HTTP/1.1 (IHttpMinRequestBodyDataRateFeature). This is a pretty big hammer to wield though. Lets discuss in person.

Reply:
As per offline discussion, we'll need to change the underlying data rate mechanism in Kestrel. I'll file an issue.

Reply:
I've updated the tracking issue. This will need a feature from Kestrel: https://github.com/aspnet/AspNetCore/issues/7635. In the meantime for this release, I'll remove this setting and @shirhatti will add a note in the blog post to clarify how to enable the client streaming scenarios.